infrastructure/simulation-tool/src/utils/dev.utils.ts

```diff
@@ -0,0 +1,37 @@
+import { providers, Signer } from 'ethers';
+
+const prepareDevL1Account = async (account: Signer): Promise<providers.TransactionReceipt> => {
+    console.log('Finding first account with some balance ...');
+    const provider = account.provider as providers.JsonRpcProvider;
+    const devAccounts = await provider.listAccounts();
+    const loadedSigner = devAccounts
+        .map((accountAddress: string) => provider.getSigner(accountAddress))
+        .find(async (signer: Signer) => {
+            let balance = await signer.getBalance();
+
+            return !balance.isZero();
+        });
+
+    if (!loadedSigner) {
+        throw Error(
+            'The provider does not contain a list of wealthy dev accounts. Cannot continue, please check your hardhat configuration.'
+        );
+    }
+
+    const latestBlock = await provider.getBlock('latest');
+    const gasLimit = latestBlock.gasLimit;
+    console.log(`Found ${loadedSigner._address}. Syphoning ...`);
+
+    const txRequest = await loadedSigner.populateTransaction({
+        to: await account.getAddress(),
+        value: (await loadedSigner.getBalance()).sub(gasLimit),
+        gasLimit
```

Comment:
Not sure why we're setting the `gasLimit` here

Reply:
I think I've done that to allow enough value to cover gas costs. The gas limit is somewhat arbitrary.

Reply:
removed the gas limit option